### PR TITLE
feat: subset translation

### PR DESCRIPTION
**Pull request by tokers**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests

- Target branch

### CODE REVIEW

File: pkg/ingress/pod.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+func (c *Controller) newPodController() *podController {
+	ctl := &podController{
+		controller: c,
+	}
+	ctl.controller.podInformer.AddEventHandler(
+		cache.ResourceEventHandlerFuncs{
+			AddFunc:    ctl.onAdd,
+			DeleteFunc: ctl.onDelete,
```
Comment (gxthrj) [inline pkg/ingress/pod.go:38]: There is  no updateFunc?

File: pkg/ingress/pod.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+		return
+	}
+	if !c.controller.namespaceWatching(key) {
+		return
+	}
+	log.Debugw("pod add event arrived",
+		zap.Any("object", obj),
+	)
+	pod := obj.(*corev1.Pod)
+	if err := c.controller.podCache.Add(pod); err != nil {
```
Comment (gxthrj) [inline pkg/ingress/pod.go:69]: Need to check the status of pod, on some status like pending, the pod IP is not exist.

File: pkg/ingress/pod.go (hunk 1 above)
Comment (tokers) [reply pkg/ingress/pod.go:38]: Don't need to handle update event, as the pod's IP  and name won't change.

File: pkg/ingress/pod.go (hunk 2 above)
Comment (tokers) [reply pkg/ingress/pod.go:69]: handled inside `podCache.Add`.

**Review by gxthrj — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**